src/planner/ostree.rs

```diff
@@ -183,6 +194,16 @@ impl Planner for Ostree {
                 .map_err(PlannerError::Action)?
                 .boxed(),
         );
+
+        if Path::new("/etc/fish").is_directory() {
+            plan.push(
+                CreateDirectory::plan(&OSTREE_FISH_PROFILE_LOCATION, None, None, 0o0755, true)
```

Comment:
This uses `create_dir`, which doesn't create non-existent parents and errors if the path already exists.

Reply:
It should not error if the path exists, so long as permissions are right: https://github.com/DeterminateSystems/nix-installer/blob/a026cc2224ec2cc7aa5d53c542521eb0cea1ad37/src/action/base/create_directory.rs#L98